tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordSession.cs

```diff
@@ -102,6 +95,19 @@ public RecordEntry Lookup(RecordEntry requestEntry, RecordMatcher matcher, IEnum
             {
                 sanitizer.Sanitize(requestEntry);
             }
+            return LookUpSanitizedRecord(requestEntry, matcher, remove);
```

Comment:
Merge methods?

Reply:
Now that normalization is a single line.

Reply:
they have different signatures, I suppose we can just delete PlaybackTransport so we only would need the one signature.